GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.ts

```diff
@@ -859,10 +859,12 @@ namespace gdjs {
         const electron = this.getElectron();
         if (electron) {
           electron.shell.openExternal(url);
-        } else {
           // @ts-ignore
-          const target = window.cordova ? '_system' : '_blank';
-          window.open(url, target);
+        } else if (typeof window.cordova !== 'undefined') {
```

Comment:
To be 100% safe, might be worth checking if `typeof window.cordova.InAppBrowser !== 'undefined'`?